src/components/Auth/Signup.jsx

```diff
@@ -0,0 +1,209 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Form, FormGroup, Label, Input,
+} from 'reactstrap';
+import { Link } from 'react-router-dom';
+import Side from './Side';
+import img from '../../assets/developers_2_A1_Rectangle_58_pattern.png';
+import { userRegister, registerLoad } from '../../redux/action-creators/signup';
+
+class Register extends Component {
+  state = {
+    username: '',
+    password1: '',
+    password2: '',
+    email: '',
+
+  };
+
+  onChange = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+  }
+
+  onSubmit = (e) => {
```

Comment:
Also, 
```javascript 
onSubmit = e => {}
 ```
can also work.
In js, arrow function, the example below cannot work 
```javascript
arrowFunctionCannotWork = one, two => {}
```
When parameters are more than 2, then we can spread them or add the brackets. 
```javascript
thisWork = (one, two, three) => {} 
```

Reply:
Thank you @me-x-mi I know it is a good practice but the way our `ESLINT` is configured, it throws a warning when you don't wrap that in parenthesis.